src/test/shared/treeview/testUtil.ts

```diff
@@ -0,0 +1,43 @@
+/*!
```

Comment:
Maybe all of the "test util" file should live in once place, could be `src/test/shared/` for now. Else it's hard to discover what's available, unless one guesses that a project-wide search for "testUtil" will get most of them.

Reply:
Yeah combining all the important functions into a single file could work out ok if given more verbose names. Might look a bit messy though. Could also re-export from a single file?

Reply:
Not a single file, just a single folder :) Right now, we have testUtils in various folders.

Reply:
I'll follow up on this in a separate PR.